docs/perf_analyzer.md

```diff
@@ -400,3 +400,39 @@ can achieve by using shared memory in your application. Use
 By default perf_analyzer uses HTTP to communicate with Triton. The GRPC
 protocol can be specificed with the -i option. If GRPC is selected the
 --streaming option can also be specified for GRPC streaming.
+
+## Using C API directly
+
+### Prerequisite
+Have the Triton SDK container and the Inference Server container on your machine
+Since you will need access to the Tritonserver install, it might be easier if 
+you copy the perf_analyzer binary to the Inference Server container.
+
+### Required Params
+Use the --help option to see complete documentation for how to use the C API.
+By default perf_analyzer relies on the triton instance running. You can select
```

Comment:
By default perf_analyzer expects the Triton instance to already be running. You can configure the C API mode using the --service-kind option